src/identify/consts.js

```diff
@@ -2,5 +2,9 @@
 
 module.exports.PROTOCOL_VERSION = 'ipfs/0.1.0'
 module.exports.AGENT_VERSION = 'js-libp2p/0.1.0'
-module.exports.MULTICODEC_IDENTIFY = '/ipfs/id/1.0.0'
-module.exports.MULTICODEC_IDENTIFY_PUSH = '/ipfs/id/push/1.0.0'
+module.exports.MULTICODEC_IDENTIFY = '/p2p/id/1.1.0'
+module.exports.MULTICODEC_IDENTIFY_PUSH = '/p2p/id/push/1.1.0'
```

Comment:
Where are the versions listed, I was having trouble finding the updates. Is this in the specs repo or in go-libp2p?

Reply:
This commit added them: https://github.com/libp2p/go-libp2p/commit/077a81814f4357379d11dc93d40f48788e515fa0#diff-98be6dba669aa4123b7ef190fe7113e8

However, now that I looked further, the same protocol is now being used and go reverted this change per https://github.com/libp2p/go-libp2p/pull/907#discussion_r419878500

I am doing the same